test/everypolitician/popolo/event_test.rb

```diff
@@ -27,4 +31,24 @@ def test_accessing_event_properties
     assert_equal '2014-11-27', event.start_date
     assert_nil event.end_date
   end
+
+  def test_event_people_returns_array_of_person_objects
+    event = popolo_wales.events.map { |e| e }[4]
+    assert_instance_of Everypolitician::Popolo::Person, event.people.first
+  end
+
+  def test_event_organizations
+    event = popolo_wales.events.map { |e| e }[4]
+    assert_instance_of Everypolitician::Popolo::Organization, event.organizations.first
+  end
+
+  def test_event_people_raises_exception
+    event = popolo_wales.events.first
+    assert_raises(NotImplementedError) { event.people.first }
+  end
```

Comment:
This test is quite confusing, both in name and in implementation. You've just shown that `event.people` is meant to return something, so why is this version blowing up? Both the name of the test, and the implementation should make that obvious (from reading some more I'm now assuming that this one is because it's an Election, whereas this only works on Terms — if that's so then the test should say so explicitly)